src/app/clusters/network-commissioning/network-commissioning.h

```diff
@@ -100,6 +100,10 @@ class Instance : public CommandHandlerInterface,
     uint8_t mLastNetworkID[DeviceLayer::NetworkCommissioning::kMaxNetworkIDLen];
     uint8_t mLastNetworkIDLen = 0;
 
+    Optional<uint64_t> mCurrentOperationBreadcrumb;
+    void UpdateBreadcrumb(const Optional<uint64_t> & breadcrumbValue);
+    void UpdateBreadcrumb();
```

Comment:
This needs some comments or different naming. The `UpdateBreadcrumb` with an argument takes in an optional, so for me as a reader of an API it is unclear how to distinguish between `UpdateBreadcrumb(Optional::Missing())` and `UpdateBreadcrumb()`.

Reply:
Renamed the one without arguments to `CommitSavedBreadcrumb`